godot/components/cooking/ApplianceInteractable.gd

```diff
@@ -0,0 +1,14 @@
+extends Interactable2D
```

Comment:
I'm probs missing something, but why not have Appliance `extends Interactable2D` directly?

Reply:
i had this originally but wanted to keep it separate for the flexibility, so that i could have one node just focused on the interactable part and another node focused solely on the appliance part

Reply:
I don't think this is necessary since the Interactable2D is designed to be extended since it only has function definitions. It also makes sense when you consider that interacting with the appliance is the driver for all of the appliance's functionality, so an appliance cannot exist with being interactable. I'd recommend switching it back to a single class for simplicity